server/src/main/java/com/objectcomputing/checkins/services/checkins/CheckInServicesImpl.java

```diff
@@ -18,62 +20,80 @@
 
     @Inject
     private MemberProfileRepository memberRepo;
-    
+
+    @Inject
+    SecurityService securityService;
+
     @Override
     public CheckIn save(CheckIn checkIn) {
         CheckIn checkInRet = null;
-        if(checkIn!=null){
+
+        if(checkIn!=null) {
```

Comment:
Is there a scenario where we want the client to be able to attempt to save nothing and simply allow that to pass? Would it make more sense to use the @NotNull annotation and simply not allow this field to be null?

Reply:
Good catch. I'll make the required changes.